ngraph/CMakeLists.txt

```diff
@@ -89,8 +89,12 @@ option(NGRAPH_UB_SANITIZER_ENABLE "Compiles and links with Undefined Behavior Sa
 option(NGRAPH_USE_PROTOBUF_LITE "Compiles and links with protobuf-lite" OFF)
 
 if (NGRAPH_ONNX_IMPORT_ENABLE)
+    option(NGRAPH_ONNX_EDITOR_ENABLE "Enable ONNX Editor" OFF)
```

Comment:
I think we can just set this to OFF by default, without the `if` condition here.

Reply:
done